src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -2,7 +2,14 @@
 
 ## Unreleased
 
-* Made `OpenTelemetry.Exporter.OtlpLogExporter` public. ([#4979](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4979))
+* Made `OpenTelemetry.Exporter.OtlpLogExporter` public.
+  ([#4979](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4979))
+
+* Updated the `OpenTelemetryLoggerOptions.AddOtlpExporter` extension to retrieve
```

Comment:
Mention the new extension method?

Reply:
I don't think we need to. It's an implementation detail as far as the extension is concerned. What I have here is the "what" changed not so much the "how" it was done.

It is mentioned in the SDK CHANGELOG though.

Reply:
I am referring to https://github.com/open-telemetry/opentelemetry-dotnet/blob/8cd1574fd783f8ed7f756808392b12ba4508ea61/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpLogExporterHelperExtensions.cs#L62

Reply:
Ah gotcha. Done!